src/target/riscv/riscv-013.c

```diff
@@ -2065,22 +2135,113 @@ static int read_memory(struct target *target, target_addr_t address,
 		uint32_t size, uint32_t count, uint8_t *buffer)
 {
 	RISCV013_INFO(info);
-	if (info->progbufsize >= 2) {
+	if (info->progbufsize >= 2)
 		return read_memory_progbuf(target, address, size, count, buffer);
```

Comment:
is no brackets on single-line ifs part of the openocd coding standards?

Reply:
Indeed.